banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/bank_reconciliation_tool_beta.js

```diff
@@ -51,50 +51,60 @@ frappe.ui.form.on('Bank Reconciliation Tool Beta', {
 		frm.disable_save();
 		frm.fields_dict["filters_section"].collapse(false);
 
-		frm.add_custom_button(__("Get Bank Transactions"), function() {
-			if (!frm.doc.bank_account) {
-				frappe.throw(
-					{
-						message: __("Please set the 'Bank Account' filter"),
-						title: __("Filter Required")
-					}
-				);
-			}
-
-			frm.events.add_upload_statement_button(frm);
```

Comment:
This was added here on clicking "Get Bank Transactions" to avoid this:
<img width="1361" alt="Screenshot 2023-10-05 at 5 23 23 PM" src="https://github.com/alyf-de/banking/assets/25857446/d8bc771c-3e4e-4b49-a9c1-2b0c8f4b922b">

It was UX decision to first show it in the empty area as a quick action and then move it to the toolbar after transactions are fetched (the empty state changes)

Reply:
Let's just remove the button from the empty state. It's sufficient to have it in the menu.